kivy/uix/recycleboxlayout.py

```diff
@@ -91,8 +91,6 @@ def _update_sizes(self, changed):
                         xo = posx - (w / 2.)
                 opt['pos'] = [xo, yo]
 
-        return relayout
```

Comment:
Ah, yes. The method returns a boolean. It iterates to find new size and if successful, returns `True`.